src/ScriptEditor/ui/Editor.tsx

```diff
@@ -26,6 +28,12 @@ export function Editor({ onMount, onChange, onUnmount }: EditorProps) {
     // Before initializing monaco editor
     scriptEditor.initialize();
 
+    for (const server of GetAllServers()) {
+      for (const [path, script] of server.scripts.entries()) {
```

Comment:
I have some concerns about this. Does it make sense to create models for *every* script across *every* server? Although technically you can be editing code across multiple servers at once, it's a confusing and suboptimal experience. (On the other hand, have scripts on every server, due to automatic copying, is quite common.)